packages/console/src/icons/Close.tsx

```diff
@@ -1,10 +1,17 @@
 import React, { SVGProps } from 'react';
 
 const Close = (props: SVGProps<SVGSVGElement>) => (
-  <svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" {...props}>
+  <svg
+    width="18"
+    height="18"
+    viewBox="0 0 18 18"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
     <path
-      d="M7.05727 7.05703C7.57797 6.53633 8.42219 6.53633 8.94289 7.05703L16.0001 14.1142L23.0573 7.05703C23.578 6.53633 24.4222 6.53633 24.9429 7.05703C25.4636 7.57773 25.4636 8.42195 24.9429 8.94265L17.8857 15.9998L24.9429 23.057C25.4636 23.5777 25.4636 24.4219 24.9429 24.9426C24.4222 25.4633 23.578 25.4633 23.0573 24.9426L16.0001 17.8855L8.94289 24.9426C8.42219 25.4633 7.57797 25.4633 7.05727 24.9426C6.53657 24.4219 6.53657 23.5777 7.05727 23.057L14.1145 15.9998L7.05727 8.94265C6.53657 8.42195 6.53657 7.57773 7.05727 7.05703Z"
-      fill="#333333"
+      d="M10.41 9L16.71 2.71C16.8983 2.52169 17.0041 2.2663 17.0041 2C17.0041 1.7337 16.8983 1.4783 16.71 1.29C16.5217 1.10169 16.2663 0.995907 16 0.995907C15.7337 0.995907 15.4783 1.10169 15.29 1.29L9 7.59L2.71 1.29C2.5217 1.10169 2.2663 0.995907 2 0.995907C1.7337 0.995907 1.4783 1.10169 1.29 1.29C1.1017 1.4783 0.995909 1.7337 0.995908 2C0.995908 2.2663 1.1017 2.52169 1.29 2.71L7.59 9L1.29 15.29C1.19627 15.383 1.12188 15.4936 1.07111 15.6154C1.02034 15.7373 0.994202 15.868 0.994202 16C0.994202 16.132 1.02034 16.2627 1.07111 16.3846C1.12188 16.5064 1.19627 16.617 1.29 16.71C1.38296 16.8037 1.49356 16.8781 1.61542 16.9289C1.73728 16.9797 1.86799 17.0058 2 17.0058C2.13201 17.0058 2.26272 16.9797 2.38458 16.9289C2.50644 16.8781 2.61704 16.8037 2.71 16.71L9 10.41L15.29 16.71C15.383 16.8037 15.4936 16.8781 15.6154 16.9289C15.7373 16.9797 15.868 17.0058 16 17.0058C16.132 17.0058 16.2627 16.9797 16.3846 16.9289C16.5064 16.8781 16.617 16.8037 16.71 16.71C16.8037 16.617 16.8781 16.5064 16.9289 16.3846C16.9797 16.2627 17.0058 16.132 17.0058 16C17.0058 15.868 16.9797 15.7373 16.9289 15.6154C16.8781 15.4936 16.8037 15.383 16.71 15.29L10.41 9Z"
+      fill="#747778"
```

Comment:
nit: should we leave the fill empty and use CSS to control the color here?

Reply:
LOG-1912 let's align them in next weekly meeting.